txclib/parsers.py

```diff
@@ -392,11 +392,15 @@ def set_parser(subparser=False, is_legacy=False):
         "--execute", action="store_true", dest="execute", default=False,
         help="Execute commands.")
     auto_local_parser.add_argument(
-        "expression", action="store",
-        help="A path expression pointing to the location where translation "
-             "files for the associated source file are/will be saved. Use "
-             "<lang> as a wildcard for the language code, "
-             "e.g. translations/<lang>/test.txt."
+        "--expression", action="store",
+        help=("Expression defining where translation files should be saved. "
+              "Default value is: "
+              "'locale/<lang>/{filepath}/{filename}{extension}'")
```

Comment:
@rigaspapas sorry for not thinking of this earlier,  I don't think we can have a default expression for this case. The logic to replace `filename`, etc.. is implemented only in the `tx config mapping-bulk` subcommand. Since we cannot make any one of the two arguments mandatory without breaking someone's workflow I think we shouldn't use any default value just check both arguments and if non is set to through an error.

Reply:
@SofiaMargariti that sounds like a safer solution. I will update the logic.